README.md

```diff
@@ -23,7 +23,7 @@ Configure with your API token, region and optional timeout (default value is 30)
 require onfido
 
 Onfido.configure do |config|
-  config.api_token = ENV["ONFIDO_API_TOKEN"]
+  config.api_token = ENV["ONFIDO_WEBHOOK_SECRET_TOKEN"]
```

Comment:
Is this right?

Reply:
Indeed it's not right. I'll fix it.

Reply:
Fixed, I'll submit the change to the template in upcoming PR.